builder.md

```diff
@@ -63,8 +97,12 @@ Scheme files have the following structure:
     base0E: "eeeeee"
     base0F: "ffffff"
 
-- Hexadecimal color values may optionally be preceded by a "#".
-- Hexadecimal color values are case insensitive.
+When `system` is detected as `base16`:
```

Comment:
I think we can change this to say when `system` is missing, scheme files should be parsed as base16 scheme files.

Reply:
Sure.